docs/book/how-to/snapshots/snapshots.md

```diff
@@ -0,0 +1,300 @@
+---
+description: Create and run pipeline snapshots.
+icon: print
+---
+
+# Pipeline Snapshots
+
+In ZenML, pipeline snapshots are pre-defined, parameterized configurations for your pipelines that can be easily run from various interfaces - including the Python SDK, CLI, ZenML dashboard, or REST API.
+
+{% hint style="info" %}
+Snapshots are the successor and replacement of ZenML run templates.
+{% endhint %}
+
+{% hint style="success" %}
+Running snapshots is a [ZenML Pro](https://zenml.io/pro)-only feature.
+{% endhint %}
+
+## Real-world Use Case
+
+Imagine your team has built a robust training pipeline that needs to be run regularly with different parameters:
+
+- **Data Scientists** need to experiment with new datasets and hyperparameters
+- **MLOps Engineers** need to schedule regular retraining with production data
+- **Stakeholders** need to trigger model training through a simple UI without coding
+
+Without snapshots, each scenario would require:
+1. Direct access to the codebase 
+2. Knowledge of pipeline implementation details
+3. Manual pipeline configuration for each run
+
+**Pipeline snapshots solve this problem by creating a reusable configuration** that can be executed with different parameters from any interface:
+
+- **Through Python**: Data scientists can programmatically trigger snapshots with custom parameters
+```python
+  from zenml.client import Client
+  
+  Client().trigger_pipeline(
+      snapshot_name_or_id=<NAME-OR-ID>,
+      run_configuration={
+          "steps": {
+              "data_loader": {"parameters": {"data_path": "s3://new-data/"}},
+              "model_trainer": {"parameters": {"learning_rate": 0.01}}
+          }
+      }
+  )
+
+- **Through REST API**: Your CI/CD system can trigger snapshots via API calls
+  ```bash
+  curl -X POST 'https://your-zenml-server/api/v1/pipeline-snapshots/<ID>/runs' -H 'Authorization: Bearer <TOKEN>' -d '{"steps": {...}}'
+  ```
```

Comment:
I think these have to be flush against the left for gitbook to parse.

```suggestion
```bash
  curl -X POST 'https://your-zenml-server/api/v1/pipeline-snapshots/<ID>/runs' -H 'Authorization: Bearer <TOKEN>' -d '{"steps": {...}}'
```
```